src/test/java/uk/gov/hmcts/probate/service/NotificationServiceTest.java

```diff
@@ -1642,6 +1709,18 @@ public void sendGeneralCaveatExtendEmail()
         personalisation.put(PERSONALISATION_REGISTRY_PHONE, "0300 303 0648");
         personalisation.put(PERSONALISATION_CAVEAT_EXPIRY_DATE, "1st January 2019");
         personalisation.put(PERSONALISATION_WELSH_CAVEAT_EXPIRY_DATE, "1 Ionawr 2019");
+        personalisation.put(PERSONALISATION_WELSH_DATE_OF_DEATH, "12 Rhagfyr 2000");
+        personalisation.put(PERSONALISATION_DATE_OF_DEATH, "12th December 2000");
+        if(caveatData.getDeceasedDateOfBirth() != null){
```

Comment:
Do you need both ENG and WEL here? Does the template need both? I would have thought the template only needs one or other - in which case you should be able to determine ENG/WEL from the case data?